plugins/frontend/backstage-plugin-github-insights/src/mocks/mocks.ts

```diff
@@ -420,7 +420,7 @@ export const releasesResponseMock = [

```

Comment:
It may have been unnecessary for me to change these mocked values as well. Please let me know if you would like me to revert this. Thanks!